### PR TITLE
feat: add slide toggle form field

### PR DESCRIPTION
**Pull request by lab900-frederico-teixeira**
## Purpose

To add [slide toggle](https://material.angular.io/components/slide-toggle/overview) component as a form field

### CODE REVIEW

**Review by lab900-senn-lambrecht — APPROVED** [review]

File: lib/src/lib/components/form-fields/slide-toggle-field/slide-toggle-field.component.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  *ngIf="!fieldIsHidden"
+  class="lab900-slide-toggle-field mat-form-field-wrapper"
+  id="lab900-slide-toggle-field-{{ schema?.attribute }}"
+>
+  <div *ngIf="schema.title" class="lab900-slide-toggle-field__title">
+    <mat-label>{{ schema.title | translate }}</mat-label>
+  </div>
+  <mat-slide-toggle
+    [disabled]="fieldIsReadonly"
+    [formControlName]="schema.attribute"
```
Comment (lab900-senn-lambrecht) [inline lib/src/lib/components/form-fields/slide-toggle-field/slide-toggle-field.component.html:12]: replace by `fieldAttribute`

File: lib/src/lib/components/form-fields/slide-toggle-field/slide-toggle-field.component.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<div
+  [formGroup]="group"
+  *ngIf="!fieldIsHidden"
+  class="lab900-slide-toggle-field mat-form-field-wrapper"
+  id="lab900-slide-toggle-field-{{ schema?.attribute }}"
```
Comment (lab900-senn-lambrecht) [inline lib/src/lib/components/form-fields/slide-toggle-field/slide-toggle-field.component.html:5]: replace by `fieldAttribute`

**Review by lab900-senn-lambrecht — CHANGES_REQUESTED** [review]

File: lib/src/lib/components/form-fields/slide-toggle-field/slide-toggle-field.component.html (hunk 1 above)
Comment (lab900-frederico-teixeira) [reply lib/src/lib/components/form-fields/slide-toggle-field/slide-toggle-field.component.html]: done

File: lib/src/lib/components/form-fields/slide-toggle-field/slide-toggle-field.component.html (hunk 2 above)
Comment (lab900-frederico-teixeira) [reply lib/src/lib/components/form-fields/slide-toggle-field/slide-toggle-field.component.html]: done

**Review by lab900-senn-lambrecht — APPROVED** [review]

**Pull request merged**